tf_encrypted/protocol/aby3/aby3.py

```diff
@@ -2559,30 +2532,16 @@ def _B_ppa_sklansky_private_private(prot, x, y, n_bits):
         x.backing_dtype.native_type))
 
   with tf.name_scope("B_ppa"):
-    if prot.nbits == 64:
-      keep_masks = [
-          0x5555555555555555, 0x3333333333333333,
-          0x0f0f0f0f0f0f0f0f, 0x00ff00ff00ff00ff,
-          0x0000ffff0000ffff, 0x00000000ffffffff
-      ]  # yapf: disable
-      copy_masks = [
-          0x5555555555555555, 0x2222222222222222,
-          0x0808080808080808, 0x0080008000800080,
-          0x0000800000008000, 0x0000000080000000
-      ]  # yapf: disable
-    elif prot.nbits == 128:
```

Comment:
out of curiosity, did you remove this because you never needed 128 bit numbers?

Reply:
> out of curiosity, did you remove this because you never needed 128 bit numbers?

Right. It was tested before, but not completely implemented in this version. Better remove it so it does not confuse the users.